server/src/main/resources/db/V10__create_guilds_table.sql

```diff
@@ -0,0 +1,15 @@
+drop table if exists guilds;
```

Comment:
@chrismckiernan there is V10 in develop so please change it to V11 and other one to V12. Also if these are in its own PR then chances are they will be merged quickly and we will not have another case of missing versions.

Reply:
Done